models/edr/run_results/model_run_results.sql

```diff
@@ -1,3 +1,10 @@
+{{
```

Comment:
This is a comment for the folders change above - why elementary_test_results is now in run_results?